go.mod

```diff
@@ -1,41 +1,34 @@
 module github.com/openshift/rbac-permissions-operator
 
-go 1.17
+go 1.19
 
 require (
 	github.com/golang/mock v1.6.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/gomega v1.24.1
 	github.com/openshift/operator-custom-metrics v0.4.3-0.20220322205053-7b528cc0d6eb
 	github.com/operator-framework/operator-lib v0.10.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.62.0
-	github.com/prometheus/client_golang v1.12.2
+	github.com/prometheus/client_golang v1.14.0
 	github.com/sykesm/zap-logfmt v0.0.4
-	go.uber.org/zap v1.21.0
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/client-go v12.0.0+incompatible
```

Comment:
I remember we have reason to keep this for that version?

Reply:
I don't remember what reason we had for that, but it seems to build, compile and work with 0.26.1.